.github/workflows/tests.yml

```diff
@@ -16,12 +16,12 @@ jobs:
         include:
           - php-version: 7.1
             symfony-version: 4.4.*
-          - php-version: 7.4
+          - php-version: 8.0
```

Comment:
Why did you remove the support for php 7.4?

Reply:
The bundle is still compatible with PHP 7.4, this is all about having a shorter testing matrix : only lowest and highest versions tested

Do you think it can be error prone ?

Reply:
Not at all, I was just wondering why you were removing that version which I was considering as a major version.

However I like keeping only the lowest and highest versions. Or maybe you could keep the lowest and highest for each major? Like `7.1` and `7.4`, and `8.0`... until the next minor is released?

Reply:
Actually, there is no need to keep `7.4` in testing matrix, the idea was to keep lowest and highest PHP version for each lowest and highest Symfony version